### PR TITLE
Added JAR for Mule 4

### PR DESCRIPTION
**Pull request by hannanelson**
Clarified that mule-app.properties is Mule 3 and YAML is Mule 4.

### CODE REVIEW

**Review by rishath-rias — COMMENTED** [review]
The Mule 4 text for JAR looks good to me. I'm not sure if the 'Secure Properties Placeholder' which was part of Security Module in Mule 3 actually gets bundled in Mule 3 zip file. Can we get that verified. I tried looking for it, but couldn't find conclusive statements on Mule 3.

File: modules/ROOT/pages/cloudhub-manage-props.adoc [hunk 1]
```diff
@@ -65,30 +65,33 @@ To create set an environment variable using an application property:
...
+See xref:mule-runtime::configuring-properties.adoc[Configuring Properties].
+* `mule-app.properties` (Mule 3)
++
+See xref:3.9@mule-runtime::configuring-properties.adoc[Configuring Properties].
 
-== Add Application Properties Using the mule-app.properties File
+CloudHub then loads these properties into the application when the application starts. 
 
-You can add a `mule-app.properties` file inside the deployable application archive file. CloudHub then loads these properties into the application when the application starts. 
+*IS THIS TRUE?*
```
Comment (MichelleZMY) [inline modules/ROOT/pages/cloudhub-manage-props.adoc]: yes!

**Review by MichelleZMY — APPROVED** [review]
LGTM

File: modules/ROOT/pages/cloudhub-manage-props.adoc [hunk 2]
```diff
@@ -9,9 +9,9 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybr
 image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
 image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
 
-In a CloudHub deployment, you can add properties to an application using Anypoint Runtime Manager or the `mule-app.properties` file.
+In a CloudHub deployment, you can add properties to an application using Anypoint Runtime Manager or a properties file.
```
Comment (kbostrom) [inline modules/ROOT/pages/cloudhub-manage-props.adoc]:
```suggestion
In a CloudHub deployment, you can add properties to an application by using either Anypoint Runtime Manager or a properties file.
```

File: modules/ROOT/pages/cloudhub-manage-props.adoc [hunk 3]
```diff
@@ -9,9 +9,9 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybr
...
 
-If you set properties both in `mule-app.properties` in your application and in Runtime Manager, the values set in Runtime Manager always override the values in `mule-app.properties` when your application deploys.
+If you set properties both in the properties file in your application and in Runtime Manager, the values set in Runtime Manager always override the values in properties file when your application deploys.
```
Comment (kbostrom) [inline modules/ROOT/pages/cloudhub-manage-props.adoc]:
```suggestion
If you set properties both in the properties file in your application and in Runtime Manager, the values set in Runtime Manager always override the values in the properties file when your application deploys.
```

File: modules/ROOT/pages/hybrid-manage-props.adoc [hunk 4]
```diff
@@ -11,7 +11,7 @@ image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoin
 
 
 
-In an on-premises (hybrid or Anypoint Platform PCE) deployment, you can add properties to an application using the `mule-app.properties` file or Anypoint Runtime Manager.
+In an on-premises (hybrid or Anypoint Platform PCE) deployment, you can add properties to an application using a properties file or Anypoint Runtime Manager.
```
Comment (kbostrom) [inline modules/ROOT/pages/hybrid-manage-props.adoc]:
```suggestion
In an on-premises (hybrid or Anypoint Platform PCE) deployment, you can add properties to an application by using either a properties file or Anypoint Runtime Manager.
```

File: modules/ROOT/pages/hybrid-manage-props.adoc [hunk 5]
```diff
@@ -29,16 +29,22 @@ The following security considerations apply to properties:
 * Runtime Manager displays the values of the properties only in the *Settings* page.
 
 
-== Add Application Properties Using the mule-app.properties File
+== Add Application Properties Using a Properties File
```
Comment (kbostrom) [inline modules/ROOT/pages/hybrid-manage-props.adoc]:
```suggestion
== Add Application Properties by Using a Properties File
```

**Review by kbostrom — APPROVED** [review]
Edited

**Pull request merged**